model/task_lifecycle.go

```diff
@@ -1313,7 +1428,8 @@ func evalBisectStepback(ctx context.Context, t *task.Task, caller string, newSte
 	}
 
 	existingStepback := !t.StepbackInfo.IsZero() && t.ActivatedBy == evergreen.StepbackTaskActivator
-	if newStepback || existingStepback {
+	generatedTask := t.GeneratedBy != ""
```

Comment:
Rather than just checking if this is a generated task, I think we should check if this is a generated task _and_ its generator task has an existing stepback going for this task's BV/Display name pair.